server/queue/queue-activemq/src/main/java/org/apache/james/queue/activemq/FileSystemBlobStrategy.java

```diff
@@ -58,40 +59,20 @@ public FileSystemBlobStrategy(BlobTransferPolicy policy, FileSystem fileSystem,
 
     @Override
     public URL uploadFile(ActiveMQBlobMessage message, File file) throws JMSException, IOException {
-        return uploadStream(message, new FileInputStream(file));
+        try (FileInputStream in = new FileInputStream(file)) {
+            return uploadStream(message, in);
+        }
     }
 
     @Override
     public URL uploadStream(ActiveMQBlobMessage message, InputStream in) throws JMSException, IOException {
```

Comment:
we previously closed the inputstream after using it and it's no longer the case.
Either we checked every callers of uploadStream or we should not change that behavior